fbu-app/ViewControllers/FeatureScreens/BulletinViewController.m

```diff
@@ -0,0 +1,70 @@
+//
+//  BulletinViewController.m
+//  fbu-app
+//
+//  Created by lucjia on 8/1/19.
+//  Copyright © 2019 lucjia. All rights reserved.
+//
+
+#import "BulletinViewController.h"
+#import "PostCell.h"
+#import "Parse/Parse.h"
+
+@interface BulletinViewController () <UICollectionViewDelegate, UICollectionViewDataSource> {
+    NSMutableArray *posts;
+}
+
+@property (weak, nonatomic) IBOutlet UIImageView *backgroundImage;
+@property (weak, nonatomic) IBOutlet UICollectionView *collectionView;
+
+@end
+
+@implementation BulletinViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    // set background image
+    self.backgroundImage.image = [UIImage imageNamed:@"corkboard"];
+    self.backgroundImage.clipsToBounds = YES;
+    self.backgroundImage.alpha = 0.8;
+    
+    self.collectionView.delegate = self;
+    self.collectionView.dataSource = self;
+    
+    [self fetchPosts];
+}
+
+- (void) fetchPosts {
```

Comment:
throughout our code we all have similar fetchXXX methods, we might want to make a specific class designated for this later